SocketRocket/Internal/Utilities/SRURLUtilities.m

```diff
@@ -55,7 +55,15 @@ extern BOOL SRURLRequiresSSL(NSURL *url)
             break;
         case NSURLNetworkServiceTypeVoIP: {
             networkServiceType = NSStreamNetworkServiceTypeVoIP;
-            if (NSFoundationVersionNumber > NSFoundationVersionNumber_iOS_8_3) {
+
```

Comment:
This completely changes the logic of the check here, near as I can tell. We probably shouldn't ever compare to `NSFoundationVersionNumber10_10_Max`, or `DBL_MAX` here. The entire check is to determine if `PushKit` is available, which can be done in several other ways than just a version check if we want.

The other choice being, of course, to just kill the print statement altogether, if we don't think it accomplishes anything.